main.py

```diff
@@ -221,9 +222,9 @@ def cli(dry_run, verbose):
         click.secho(
             f"\n{len(bads)} settings have a bad ratio over threshold.", fg="red"
         )
-        for source, statuses in bads:
+        for source, total, statuses in bads:
             statuses_desc = sorted(statuses, key=lambda e: e[1], reverse=True)
-            stats = " ".join([f"{s}:{v:,}" for s, v in statuses_desc if v > 0])
+            stats = " ".join([f"{s}: {v:,} {v/total*100:.2f}% " for s, v in statuses_desc if v > 0])
```

Comment:
nit: There's an extra space after the `%` character. Also, I think `black` would make this `{v / total * 100:.2f}`.